web/routing.go

```diff
@@ -138,72 +82,28 @@ func SetupRoutes(router *echo.Echo) error {
 	}
 
 	router.Use(middlewares.CORS(middlewares.CORSOptions{
-		BlackList: []string{"/auth/"},
+		BlackList: []string{},
 	}))
 
-	// non-authentified HTML routes for authentication (login, OAuth, ...)
-	{
-		mws := []echo.MiddlewareFunc{
-			middlewares.NeedInstance,
-			middlewares.LoadSession,
-			middlewares.Accept(middlewares.AcceptOptions{
-				DefaultContentTypeOffer: echo.MIMETextHTML,
-			}),
-			middlewares.CheckIE,
-			middlewares.CheckInstanceBlocked,
-		}
-		router.GET("/", auth.Home, mws...)
-		auth.Routes(router.Group("/auth", mws...))
-	}
-
-	// authentified JSON API routes
-	{
-		mwsNotBlocked := []echo.MiddlewareFunc{
-			middlewares.NeedInstance,
-			middlewares.LoadSession,
-			middlewares.Accept(middlewares.AcceptOptions{
-				DefaultContentTypeOffer: jsonapi.ContentType,
-			}),
+	/*
+		// non-authentified HTML routes
+		{
+			mws := []echo.MiddlewareFunc{
+				middlewares.Accept(middlewares.AcceptOptions{
+					DefaultContentTypeOffer: echo.MIMETextHTML,
+				}),
+				middlewares.CheckIE,
+			}
 		}
```

Comment:
Why the comments?

Reply:
It can be deleted. I kept that closely while fixing UI issues. It's useless now.